packages/core/src/etherscan.ts

```diff
@@ -116,7 +116,7 @@ export const verifyDeploymentWithRetries = async (
       let success = false
 
       const contractName = fullyQualifiedName.split(':')[1]
-      for (let attempt = 0; attempt < maxAttempts; attempt++) {
+      for (let attempt = 1; attempt <= maxAttempts; attempt++) {
```

Comment:
This line fixes the [off-by-one Etherscan log error](https://linear.app/chugsplash/issue/CHU-372/off-by-one-in-etherscan-logs-in-deploy-cli-command).